components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -7202,7 +7203,7 @@ private void handleException(String msg, Throwable t) throws APIManagementExcept
         return urlMappings;
     }
 
-    public Set<URITemplate> getURITemplatesOfAPI(APIIdentifier identifier)
+    public Set<URITemplate> getURITemplatesOfAPI(APIIdentifier identifier, String organization)
```

Comment:
this method needs to be changed to work with uuid of the api. We could remove the organization @tharikaGitHub can u check wether it is changed